client/my-sites/customer-home/main.jsx

```diff
@@ -0,0 +1,192 @@
+									action={ () => page( `/themes/${ siteSlug }` ) }
+									label={ translate( 'Change theme' ) }
+									iconSrc="/calypso/images/customer-home/theme.svg"
+								/>
+								<ActionBox
+									action={ customizeUrl }
+									label={ translate( 'Edit menus' ) }
+									iconSrc="/calypso/images/customer-home/menus.svg"
+								/>
+								<ActionBox
+									action="https://en.support.wordpress.com/images/"
+									label={ translate( 'Change images' ) }
+									iconSrc="/calypso/images/customer-home/images.svg"
+								/>
+								<ActionBox
+									action="https://logojoy.grsm.io/looka"
+									label={ translate( 'Design a logo' ) }
+									iconSrc="/calypso/images/customer-home/logo.svg"
+								/>
+								<ActionBox
+									action="https://wordpress.com/email/domain.wordpress.com"
+									label={ translate( 'Add G Suite' ) }
+									iconSrc="/calypso/images/customer-home/gsuite.svg"
+								/>
+							</div>
+						</Card>
+					</div>
+					<div className="customer-home__layout-col">
+						<Card>
+							<CardHeading>{ translate( 'Grow & Earn' ) }</CardHeading>
+							<h6 className="customer-home__card-subheader">
+								{ translate( 'Grow my audience and earn money' ) }
+							</h6>
+							<VerticalNav className="customer-home__card-links">
+								<VerticalNavItem path={ `/marketing/connections/${ siteSlug }` }>
+									{ translate( 'Share my site' ) }
+								</VerticalNavItem>
+								<VerticalNavItem path={ `/marketing/tools/${ siteSlug }` }>
+									{ translate( 'Grow my audience' ) }
+								</VerticalNavItem>
+								<VerticalNavItem path={ `/earn/${ siteSlug }` }>
+									{ translate( 'Earn money' ) }
+								</VerticalNavItem>
+							</VerticalNav>
+						</Card>
+						<Card>
+							<CardHeading>{ translate( 'Support' ) }</CardHeading>
+							<h6 className="customer-home__card-subheader">
+								{ translate( 'Get all of the help you need' ) }
+							</h6>
+							<div className="customer-home__card-support">
+								<img
+									src="/calypso/images/customer-home/happiness.png"
+									alt={ translate( 'Support' ) }
+								/>
+								<VerticalNav className="customer-home__card-links">
+									<VerticalNavItem path="https://en.support.wordpress.com/" external>
+										{ translate( 'Support docs' ) }
+									</VerticalNavItem>
+									<VerticalNavItem path="https://wordpress.com/help/contact" external>
+										{ translate( 'Contact us' ) }
+									</VerticalNavItem>
+								</VerticalNav>
+							</div>
+						</Card>
+						<Card>
+							<CardHeading>{ translate( 'Go Mobile' ) }</CardHeading>
+							<h6 className="customer-home__card-subheader">
+								{ translate( 'Make updates on the go' ) }
+							</h6>
+							<div className="customer-home__card-button-pair customer-home__card-mobile">
+								<Button
+									href="https://play.google.com/store/apps/details?id=org.wordpress.android"
+									aria-label={ translate( 'Google Play' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Get on Google Play' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).